src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/service/RecallSummaryGenerator.kt

```diff
@@ -16,15 +16,15 @@ import java.time.LocalDate
 import java.time.ZoneId
 import java.time.ZonedDateTime
 import java.time.format.DateTimeFormatter
-import java.util.Locale
+import java.util.Locale.ENGLISH
 
 @Component
 class RecallSummaryGenerator(
   @Autowired private val templateEngine: SpringTemplateEngine,
   @Autowired private val clock: Clock
 ) {
 
-  private val dateFormatter = DateTimeFormatter.ofPattern("dd MMM yyyy", Locale.ENGLISH)
+  private val dateFormatter = DateTimeFormatter.ofPattern("dd MMM yyyy", ENGLISH)
```

Comment:
Can supersede with your new `.asStandardDate`?
Also, interesting use of a locale here and not elsewhere??

Reply:
Yes I intend to reuse all over the place.  Just didn't want to change too much while others were still working in this area.

Without being specific about the Locale I was getting differences locally and in circleCI e.g. Sep vs Sept